src/main/java/by/itacademy/javaenterprise/knyazev/dao/exceptions/ExceptionDAO.java

```diff
@@ -0,0 +1,22 @@
+package by.itacademy.javaenterprise.knyazev.dao.exceptions;
+
+public abstract class ExceptionDAO extends Exception {
```

Comment:
Это `DAO` для исключений. Я так понимаю что тебе нужна ошибка, тогда правильнее будет `DAOException`
```suggestion
public abstract class DAOException extends Exception {
```